controllers/actions.github.com/ephemeralrunner_controller.go

```diff
@@ -423,12 +424,17 @@ func (r *EphemeralRunnerReconciler) cleanupRunnerLinkedSecrets(ctx context.Conte
 	return false, multierr.Combine(errs...)
 }
 
-func (r *EphemeralRunnerReconciler) markAsFailed(ctx context.Context, ephemeralRunner *v1alpha1.EphemeralRunner, log logr.Logger) error {
+func (r *EphemeralRunnerReconciler) markAsFailed(
```

Comment:
Can you please keep it in a single line? :relaxed:

Reply:
Sure!